src/api/VueSwitchbladeApi.js

```diff
@@ -104,6 +104,27 @@ class VueSwitchbladeApi {
     return this._request('/locales', { query: { language } })
   }
 
+  // Connections
+  connections () {
+    return this._request('/users/@me/connections')
+  }
+
+  getLoginURL (service) {
+    return `${this._apiURL}/connections/${service}/authURL`
+  }
+
+  async openConnectionPopup (service) {
+    window.open(
+      await this.getLoginURL(service),
```

Comment:
Why use async/await if `getLoginURL` is a synchronous method;

Reply:
fixed